lib/matplotlib/axes/_base.py

```diff
@@ -1287,25 +1287,59 @@ def apply_aspect(self, position=None):
 
     def axis(self, *v, **kwargs):
         """
-        Convenience method for manipulating the x and y view limits
-        and the aspect ratio of the plot. For details, see
-        :func:`~matplotlib.pyplot.axis`.
+        Call signature::
+
```

Comment:
All of the docstrings should be numpydoc formatted, we just have a _huge_ backlog to get through (and updating docstrings is even less sexy than triaging bugs).

Looking at this more cloesly this function does not fit well with numpydoc as it uses patterns that the numpy developers don't use.